src/components/PDFThumbnail/index.tsx

```diff
@@ -25,18 +27,31 @@ function PDFThumbnail({previewSourceURL, style, isAuthTokenRequired = false, ena
                 }}
                 externalLinkTarget="_blank"
                 onPassword={onPassword}
+                onLoad={() => {
+                    setFailedToLoad(false);
+                }}
+                onLoadError={() => {
+                    if (onLoadError) {
+                        onLoadError();
+                    }
+                    setFailedToLoad(true);
+                }}
+                error={() => null}
             >
                 <View pointerEvents="none">
                     <Thumbnail pageIndex={0} />
                 </View>
             </Document>
         ),
-        [isAuthTokenRequired, previewSourceURL, onPassword],
+        [isAuthTokenRequired, previewSourceURL, onPassword, onLoadError],
     );
 
     return (
-        <View style={[style, styles.overflowHidden]}>
-            <View style={[styles.w100, styles.h100, styles.alignItemsCenter, styles.justifyContentCenter]}>{enabled && thumbnail}</View>
+        <View style={[style, styles.h100, styles.overflowHidden]}>
```

Comment:
Here, we Added the `styles.h100` parent View Component; why is this change necessary? It was not to the parent View Component before our changes. We don't want to add any visual regression from the PR, so we wanted to make sure it is needed here